src/Components/Blueprints/BlueprintCard.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+
+import {
+  Badge,
+  Card,
+  CardHeader,
+  CardTitle,
+  CardBody,
+  CardFooter,
+  Dropdown,
+  DropdownList,
+  MenuToggle,
+  MenuToggleElement,
+  DropdownItem,
+} from '@patternfly/react-core';
+import { EllipsisVIcon } from '@patternfly/react-icons';
+
+import { BlueprintItem } from '../../store/imageBuilderApi';
+
+const BlueprintCard = (props: {
```

Comment:
The codebase's convention is to just destructure the props so we can use them directly in the component. E.g. we would simply use `blueprint` instead of `props.blueprint`.

It is also our convention to define the prop types in a separate `Type` definition right above the component. So in this case, we should create a `BluePrintCardTypes` type definition above.